api/src/main/resources/liquibase.xml

```diff
@@ -94,11 +94,16 @@
     </changeSet>
   
     <changeSet id="inslude_ABC-3TC-LPV/r_2020-09-15-14:16" author="tega" runOnChange="true">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM concept_answer WHERE concept_id='164507' and answer_concept='162200'
+            </sqlCheck>
+        </preConditions>
         <insert tableName="concept_answer">
             <column name="concept_id" valueNumeric="164507"/>
             <column name="answer_concept" valueNumeric="162200"/>
             <column name="creator" valueNumeric="1"/>
-            <column name="date_created" valueDate="now()"/>
+            <column name="date_created" valueDate="2021-02-02"/>
```

Comment:
Please change it to now(), this helps us know when the item was migrated

Reply:
This has been implemented.

Reply:
Do I need to do another pull request?

Reply:
No Bright, this one is fine, I've approved